ccd-definition/CaseEvent.json

```diff
@@ -163,7 +163,7 @@
     "Name": "Documents",
     "Description": "Upload documents",
     "DisplayOrder": 14,
-    "PreConditionState(s)": "Open;Submitted",
```

Comment:
That is regression. Please revert.

Reply:
Removed
<img width="419" alt="screenshot 2018-12-11 at 13 54 14" src="https://user-images.githubusercontent.com/42037338/49805115-444c6800-fd4c-11e8-935c-68e38d31d248.png">